docs/learn/learn-nft-pallets.md

```diff
@@ -72,9 +117,8 @@ Everything is unlocked by default (bitflag value `0`).
 
 :::info
 
-Metadata, attributes, and settings of an item or collection can be locked. The user can decide what
-to lock. Also, unauthorized and/or unprivileged transfers can be prevented by locking mechanism
-(unprivileged actions can be re-allowed anytime).
+ The user can decide to lock an item or collection’s metadata, attributes, and settings. Also, a locking mechanism can prevent unauthorized and unprivileged transfers (unprivileged
+actions can be re-allowed anytime).
```

Comment:
this paragraph seems out of place, the functionality should be described in the roles section